src/i18n/strings/en_EN.json

```diff
@@ -2527,6 +2530,7 @@
     "Option %(number)s": "Option %(number)s",
     "Write an option": "Write an option",
     "Add option": "Add option",
+    "Number of allowed selections": "Number of allowed selections",
```

Comment:
This is something that design should decide. I personally would expect something like „Number of votes per person“. It is „vote“ everywhere and not „selection“.